pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3817,11 +3817,9 @@
         "4.0.2.0": "4.3.0"
       }
     },
-    "System.Runtime.Intrinsics.Experimental": {
-      "InboxOn": {},
-      "AssemblyVersionInPackageVersion": {
-        "4.0.0.0": "4.5.0",
-        "4.0.1.0": "4.6.0"
+    "System.Runtime.Intrinsics": {
+      "InboxOn": {
+        "netcoreapp3.0": "4.0.1.0"
```

Comment:
I think this needs to go to `4.0.0.0` as well.

Reply:
Fixed.